sample-kotlin-model/src/main/kotlin/com/vimeo/dummy/sample_kotlin/KotlinSamples.kt

```diff
@@ -0,0 +1,25 @@
+package com.vimeo.dummy.sample_kotlin
+
+import com.google.gson.annotations.SerializedName
+import com.vimeo.stag.UseStag
+
+/**
+ * Test kotlin model
+ *
+ * Created by restainoa on 5/8/17.
+ */
+@UseStag
+class KotlinSamples {
+    var stringField: String? = null
+
+    var nonNullStringField: String = "default"
+
+    var intField: Int? = null
+
+    var longField: Long = 1
+
+    @SerializedName("boolean_field")
+    var booleanField: Boolean? = null
+
+    var testField: Any? = null
```

Comment:
It sounds like this would be an additional feature. And also something to look out for (or ask Brent about) if we start using Kotlin in production. It sounds like you'd need a lot of boilerplate code (generating private setters for every model) to make a model immutable.